visual-testing-app/src/App.tsx

```diff
@@ -46,40 +47,42 @@ const App = () => {
     <>
       <ThemeProvider />
       <Router>
-        <Switch>
-          <Route
-            path="/"
-            exact
-            component={() => (
-              <div>
-                <h1>Visual Testing App</h1>
-                <ul>
-                  {allSortedComponents.map((Component) => (
-                    <li key={Component.routePath}>
-                      <a href={Component.routePath}>{Component.routePath}</a>
-                    </li>
-                  ))}
-                </ul>
-              </div>
-            )}
-          />
-          {allSortedComponents.map((Component) => (
+        <CompatRouter>
```

Comment:
Other than installing the package, adding the `CompatRouter` is the only setup that needs to happen to be able to use both v5 and v6 API's.